c10/core/TensorImpl.h

```diff
@@ -1362,7 +1362,9 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
             "required rank 5 tensor to use channels_last_3d format");
         TORCH_CHECK(false, "unsupported memory format ", memory_format);
         //TODO Implement set_sizes_and_strides for channels last 3d
-        break;
+        // Cleaning warning messages, no need to break as TORCH_CHECK(false)
```

Comment:
couldn't you just remove these lines?

Reply:
Removing them is kind of error prone, if someone decide to lift "unsupported mem format" restriction.